main.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+
+
+int main(int argc, char *argv[]) {
+    // std::ofsteam configfile;
+   // configfile.open("known_recipients.txt");
+
+    //std::string argument1 = "";
+
+//std::string user_input = "";
+
+//Location
+
+//if(argument = init)
+    std::string first_and_last ="", email = "", cypher = "", timezone, path ="";
+
+    std::cout << "Enter your first and last name, please" << std::endl;
+    std::getline(std::cin, first_and_last);
+
+        while(first_and_last == ""){
+        std::cout << "Please enter your first and last name to continue" << std::endl;
```

Comment:
I did in my second one. See "Second Commit". Trying to debug my edit part now for this. and get it up, but wanted to make sure the first part was correct first and could be commented on.